### PR TITLE
Add gradient clipping example for stable neural network training

### PR DESCRIPTION
**Pull request by shyam-vyawahare**

### CODE REVIEW

**Comment by shyam-vyawahare** [comment]
Practiced & Added gradient clipping example for stable neural network training
